frontend/pages/index.tsx

```diff
@@ -33,12 +50,59 @@ export default function Home(): React.ReactNode {
           Dialog!
         </Link>
       </Heading>
-      <p>{formatMessage(messages.hello)}</p>
-      <p>{formatMessage(messages.welcomeMessage)}</p>
+      <p>{f("hello")}</p>
+      <p>{f("welcomeMessage")}</p>
       <p>Current locale: {locale}</p>
       <p>Default locale: {defaultLocale}</p>
       <p>Configured locales: {JSON.stringify(locales)}</p>
 
+      <main className={styles.main}>
+        <Box bg="brand.100" w="100%" h="100%" p={10}>
```

Comment:
its currently showing up like this on my screen: 
![Capture](https://user-images.githubusercontent.com/45128231/103180746-c68e2300-4866-11eb-907a-e69fa66fec5c.PNG)

Try setting it to width to "100vw" and height to "100vh",
100% for width and height means that it occupies 100% of the height/width of its parent container.

Reply:
Something I noticed was that we wrap this component with the `main` tag twice, once in `frontend/pages/_app.tsx line 30` as well as on `frontend/pages/index.tsx line 59`. We can probably remove the one in index.tsx since the PageWrapper handles the heavy lifting.